api/handlers/changes.js

```diff
@@ -224,12 +224,21 @@ const getChanges = feed => {
           });
           return;
         }
-        const changes = mergeChangesResponses(responses);
-        if (!changes) {
+        const malformedResponse = responses.find(response => {
+          if (!response || !response.results) {
+            // See: https://github.com/medic/medic-webapp/issues/3099
+            // This should never happen, but apparently it does sometimes.
+            // Attempting to log out the response usefully to see what's occuring
+            console.error('No _changes error, but malformed response:', JSON.stringify(changes));
```

Comment:
I'm no linter, but `changes` appears to be referenced before it's declared.

Reply:
It turns out you're a better linter than jshint... I've changed `changes` to `responses`.